.simplecov

```diff
@@ -0,0 +1,6 @@
+SimpleCov.start do
+  merge_timeout 3600
+end
+
+require 'codecov'
+SimpleCov.formatter = SimpleCov::Formatter::Codecov
```

Comment:
consider only enabling this for CI so that it doesn't bark at you about failing to upload to Codecov each time the tests are run locally:

```
if ENV['CI'] == 'true'
  require 'codecov'
  SimpleCov.formatter = SimpleCov::Formatter::Codecov
else
  SimpleCov.formatter = SimpleCov::Formatter::QualityFormatter
end
```

Reply:
Good idea. Thanks!